docs/source/examples/image-classification.rst

```diff
@@ -2,5 +2,186 @@
 
-See :ref:`quick_start:Quick Start` for the use of Image datasets
-with :ref:`supported_label_types:Classification` labels.
+| **Classification** (:ref:`ref <supported_label_types:Classification>`) is a kind of label type.
+  It represents the category of an image.
+| Here, we take `Dogs vs. Cats`_ as an example to describe
+  how to create, upload and read dataset with classification label.
+
+.. _dogs vs. cats: https://www.graviti.cn/open-datasets/DogsVsCats
+
+**********************
+ Create Local Dataset 
+**********************
+
+It takes two steps to create a dataset from local dataset files:
+
+- Write the **catalog** (:ref:`ref <basic_concepts:catalog & subcatalog>`).
+- Write the **dataloader**.
+
+Write the Catalog
+=================
+
+**Catalog** (:ref:`ref <basic_concepts:catalog & subcatalog>`)
+is the json file contains all label information of one dataset.
+
+.. literalinclude:: ../../../tensorbay/opendataset/DogsVsCats/catalog.json
+   :language: json
+   :name: dogsvscats-catalog
+   :linenos:
+
+The only label type for `Dogs vs. Cats`_ is **classification**
+(:ref:`ref <supported_label_types:Classification>`),
+and there are totally 2 **categories** (:ref:`ref <supported_label_types:Category>`)
+types ("cat" and "dog").
+
+Write the Dataloader
+====================
+
+A **dataloader** is a function to read the original dataset files into a **dataset**
+(:ref:`ref <basic_concepts:dataset>`). 
+
+.. literalinclude:: ../../../tensorbay/opendataset/DogsVsCats/loader.py
+   :language: python
+   :name: dogsvscats-dataloader
+   :linenos:
+
+It takes mainly two steps to write a **dataloader**:
+
+- Initialize **dataset** and **segments**.
+
+  - Initialize **dataset** (:ref:`ref <basic_concepts:dataset>`).(L42)
+  - Load **catalog** (:ref:`ref <basic_concepts:catalog & subcatalog>`).(L43)
+  - Create **segments** (:ref:`ref <basic_concepts:segment>`).(L46)
+
+- Initialize and append **data**.
+
+  -  Initialize **data** (:ref:`ref <basic_concepts:data>`).(L49)
+  -  Initialize **classification** (:ref:`ref <supported_label_types:Classification>`).(L51)
+  -  Write **classification** into **data**.(L51)
+  -  Append the **data** to the **segment**.(L52)
+
+See :ref:`this page <supported_label_types:Classification>` for more label details.
+
+Use Dataloaders from Community
+==============================
+
+You may also use **dataloaders** contributed by community to initialize **datasets**.
+
+.. code:: python
+
+   from tensorbay.opendataset import DogsVsCats
+
+   dataset = DogsVsCats("path/to/dataset/directory")
+
+.. note::
+
+   Note that "DogsVsCats" is not the name,
+   but the **identifier** of the dataset named "Dogs vs. Cats".
+   **Identifier** is the name of a **dataloader** function.
+
+.. warning::
+
+   Dataloaders by community work well only with the original dataset directory structure.
+   It is highly recommended to downloading datasets from either official website or
+   `Graviti Opendatset Platform`_.
```

Comment:
Graviti platfrom will not supply original zip file

Reply:
Graviti platform supplies original zip files for some datasets.